### ISSUE TITLE
Analyse fails on Xcode 10.2

### ISSUE THREAD

**Issue by ollitapa** [bitrise-steplib/steps-xcode-analyze#20]
When run with code signing disabled currently analyse runs the command

```bash
set -o pipefail && xcodebuild "-project" "SOME_APP.xcodeproj" "-scheme" "APP SCHEME" "CODE_SIGN_IDENTITY=" "CODE_SIGNING_REQUIRED=NO" "analyze" | xcpretty
```
It fails with 

```bash
❌  error: An empty identity is not valid when signing a binary for the product type 'Application'. (in target 'SOME_APP')
```

Should be 

```bash
set -o pipefail && xcodebuild "-project" "SOME_APP.xcodeproj" "-scheme" "APP SCHEME" "CODE_SIGNING_ALLOWED=NO" "analyze" | xcpretty
```

**Comment by fehersanyi-bitrise** [comment]
Hello there, we know about the issue, and already have a card for the tooling team to fix it, I hope the new step version will come out soon!

**Comment by bofeizhu** [comment]
@fehersanyi-bitrise Could we keep this open?

**Comment by sschizas** [comment]
@fehersanyi-bitrise Any update on this issue? When will this be sorted out?

**Comment by fehersanyi-bitrise** [comment]
Not yet, I will ask the team to take care of it this week. we will notify you when the fix has been deployed

**Comment by marciogranzotto** [comment]
this is a really critical issue, is breaking all the tests that we relay on

**Comment by fehersanyi-bitrise** [comment]
Hello everyone, I asked the team to take up the task as soon as they can, but unfortunately I can't do more, we will notify on every platform when the fix comes out.

**Comment by bofeizhu** [comment]
@marciogranzotto I opened this discussion on their forum two weeks ago. Still no updates. https://discuss.bitrise.io/t/xcode-analyse-fails-on-xcode-10-2/8635

**Comment by marciogranzotto** [comment]
We are 25 days in. Still no update @fehersanyi-bitrise ?
Come on, we are all developers, there's no way this is taking this long if it is a priority issue (which it should be)

**Comment by bitce** [comment]
Hi @marciogranzotto!

We are really sorry about the inconvenience this causes. There's an awful lot for our team to do and what they can and do prioritise is limited and can be defined by a lot of various aspects unfortunately. We let them know this is urgent for you and thanks for all your understanding during the wait!

**Comment by ollitapa** [comment]
Seems to have been fixed here: https://github.com/bitrise-io/go-xcode/commit/a49d262bd2151997b9b18f882654eae3aab31a9d 

I believe this repo needs just dependency update. I can try to make a pr.

**Comment by ollitapa** [comment]
Actually @BirmacherAkos has done it here already https://github.com/bitrise-steplib/steps-xcode-analyze/tree/disable_indexing

**Comment by bofeizhu** [comment]
https://github.com/bitrise-steplib/steps-xcode-analyze/pull/21

**Comment by bofeizhu** [comment]
@fehersanyi-bitrise I can confirm this issue is fixed in 2.1.0

**Comment by bofeizhu** [comment]
@fehersanyi-bitrise you can close this issue! Thank you!
